src/reticle.ts

```diff
@@ -3,6 +3,7 @@ import { StandardMaterial } from "@babylonjs/core/Materials/standardMaterial";
 import { DynamicTexture } from "@babylonjs/core/Materials/Textures/dynamicTexture";
 import { AbstractMesh } from "@babylonjs/core/Meshes/abstractMesh";
 import { CreatePlane } from "@babylonjs/core/Meshes/Builders/planeBuilder";
+import { HemisphericLight, Vector3 } from "@babylonjs/core";
```

Comment:
Import `HesiphericLight`, `Vector3` from the specific `@babylonjs/core/Lights/hemisphericLight` and `@babylonjs/core/Lights/hemisphericLight` classes in separate imports.